app/controllers/orders/new.js

```diff
@@ -15,23 +15,25 @@ export default Controller.extend({
         for (const attendee of attendees ? attendees.toArray() : []) {
           await attendee.save();
         }
-        if (paymentMode === 'free' || paymentMode === 'bank' || paymentMode === 'cheque' || paymentMode === 'onsite') {
+        if (paymentMode === 'free') {
           order.set('status', 'completed');
-        } else {
+        } else if (paymentMode === 'bank' || paymentMode === 'cheque' || paymentMode === 'onsite') {
           order.set('status', 'placed');
+        } else {
+          order.set('status', 'pending');
         }
         await order.save()
           .then(order => {
-            if (order.status === 'placed') {
+            if (order.status === 'pending') {
               this.get('notify').success(this.get('l10n').t('Order details saved. Please fill the payment details'));
-              this.transitionToRoute('orders.placed', order.identifier);
-            } else if (order.status === 'completed') {
+              this.transitionToRoute('orders.pending', order.identifier);
+            } else if (order.status === 'completed' || order.status === 'placed') {
```

Comment:
Since both of them go to same route, can we make some changes to template to differentiate between the `completed` and `placed` orders?

Reply:
Currently, this is the UI for `orders.view` route. 
![Screenshot_2019-05-20 Completed Order -e9ded88c-f393-4cb9-aeee-1d00471b69ff Orders Open Event](https://user-images.githubusercontent.com/25428397/57987717-0abe1c80-7aa3-11e9-8bde-8e9e7e3d1c02.png)

For Completed orders, we can leave it the same and for placed orders, we can replace `Your order completed successfully.` to `Your order placed successfully.`  and replace `SUCCESS` with `PLACED`. Your thought? @pradeepgangwar

Reply:
Sounds good, but i guess we need more details.

- Since placed order contain one or more kind of payment method (bank, cheque, onsite etc.), what we can do is include a comopnent below that shows the payment details i.e. What is the payment mode and how to pay for that.
- We can use following details to make user aware about how he should be paying onsite or via bank. https://github.com/fossasia/open-event-frontend/blob/0cafcc2bcddf53de0358e70b6bfaaf2f469d7bd1/app/models/event.js#L64-L67

Reply:
@pradeepgangwar it was already being displayed. I've attached images to PR description now. Please give it a look